src/slots/Slot.js

```diff
@@ -13,30 +13,10 @@ class Slot {
   /* *******************************************************************
    * UNITS
    * **************************************************************** */
-  static get UNIT_TYPES() {
-    return {
-      TEMPERATURE: 'temperature',
-      ROTATIONAL_SPEED: 'rotationalSpeed',
-      PERCENTAGE: 'percentage',
-      UNITLESS: 'unitless',
-    }
-  }
-
-  static get UNIT_TYPE_UNIT_OPTIONS() {
-    return {
-      [Slot.UNIT_TYPES.TEMPERATURE]: ['K', '°C', '°F'],
-      [Slot.UNIT_TYPES.ROTATIONAL_SPEED]: ['rpm'],
-      [Slot.UNIT_TYPES.PERCENTAGE]: ['%'],
-      [Slot.UNIT_TYPES.UNITLESS]: ['-'],
-    }
-  }
+  static get ANY_UNIT_STRING() { return 'any' }
 
-  static get UNITLESS_UNIT() { return Slot.UNIT_TYPE_UNIT_OPTIONS[Slot.UNIT_TYPES.UNITLESS][0] }
+  static get UNITLESS_UNIT() { return '-' }
 
-  static get ALL_UNIT_VALUES() {
-    return Object.values(this.UNIT_TYPE_UNIT_OPTIONS)
-      .reduce((acc, opts) => ([ ...acc, ...opts ]), [])
-  }
```

Comment:
per discussion - request to keep this in. I will make a ticket to yank out the unit registry from here so we have a central place it is kept track of.